src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -13,6 +13,19 @@
         }
       }
     },
+    "search": {
```

Comment:
Not changed as part of this PR, but the script to update the telemetry schema picked this up as something that was already missing.

Reply:
@Bamieh do you know why this could have happened?

Reply:
@Bamieh *poke* 😄

Reply:
sorry for being dormant. I think it's already been merged to master and we've reviewed to the `telemetry_check` script to make sure [it fails if the JSON file is not up-to-date](https://github.com/elastic/kibana/pull/75750) (if you rebase, it should be all good)